docs/docs/quickstart.mdx

```diff
@@ -230,9 +230,8 @@ Here's a play-by-play of what happened:
    Kratos' browser API requires a `<flow_id>` in order to log you in. It looked
    for this ID in the URL as a query parameter but couldn't find it.
 3. `/auth/login` redirected you to
-   `https://playground.projects.oryapis.com/api/kratos/public/self-service/login/browser`,
-   which is one of Ory Kratos' APIs used for logging in browser-based
-   applications.
+   `http://kratos:4434/self-service/login/browser`, which is one of Ory Kratos'
```

Comment:
```suggestion
   `http://127.0.0.1:4434/self-service/login/browser`, which is one of Ory Kratos'
```
It is only `kratos` inside of docker compose, but on you machine it will be `127.0.0.1`.